### PR TITLE
Add more hints for new contributors

### PR DESCRIPTION
**Pull request by davisagli**

### CODE REVIEW

**Review by polyester — APPROVED** [review]

**Review by stevepiercy — APPROVED** [review]

File: src/mr.roboto/src/mr/roboto/subscriber.py [hunk 1]
```diff
@@ -248,8 +248,14 @@ def run(self):
...
                 f"@{users} you need to sign the Plone Contributor "
-                f"Agreement to merge this pull request. \n\n"
+                "Agreement to merge this pull request."
+                "\n\n"
                 f"Learn about the Plone Contributor Agreement: {self.cla_url}"
+                "\n\n"
+                "If you have already signed the agreement, "
+                "please allow a week for your agreement to be processed.\n"
+                "Once it is processed, you will receive an email invitation "
+                "to join the `plone` GitHub organization as a Contributor."
```
Comment (stevepiercy) [inline src/mr.roboto/src/mr/roboto/subscriber.py:258]:
```suggestion
                "to join the `plone` GitHub organization as a Contributor."
                "If after a week, you have not received an invitation, then "
                f"please contact {self.cla_email}."
```

We also need to define `cla_email` as `agreements@plone.org` wherever `cla_url` is located.

Comment (stevepiercy) [reply src/mr.roboto/src/mr/roboto/subscriber.py:258]: Argh, and for `unknown` below, we need to add:

If you have sent in your Plone Contributor Agreement, and received an invitation to join the Plone GitHub organization, but your emails are not known, then you might need to add the email address on your Agreement to your GitHub account.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: src/mr.roboto/src/mr/roboto/subscriber.py (hunk 1 above)
Comment (davisagli) [reply src/mr.roboto/src/mr/roboto/subscriber.py:259]: @stevepiercy https://github.com/plone/mr.roboto/commit/ba516be0f173bdadb4736185cb46ca766d3be925

File: src/mr.roboto/src/mr/roboto/subscriber.py [hunk 2]
```diff
@@ -256,6 +257,9 @@ def run(self):
                 "please allow a week for your agreement to be processed.\n"
                 "Once it is processed, you will receive an email invitation "
                 "to join the `plone` GitHub organization as a Contributor."
+                "\n\n"
+                "If after a week, you have not received an invitation, then "
```
Comment (stevepiercy) [inline src/mr.roboto/src/mr/roboto/subscriber.py:261]: Sorry, I was in a rush, and my grammar was weak.

```suggestion
                "If after a week you have not received an invitation, then "
```

File: src/mr.roboto/src/mr/roboto/subscriber.py [hunk 3]
```diff
@@ -272,6 +276,11 @@ def run(self):
                 f"Learn about the Plone Contributor Agreement: {self.cla_url} "
                 f"How to add more emails to your GitHub account: "
                 f"{self.github_help_setup_email_url} "
+                "\n\n"
+                "If you have sent in your Plone Contributor Agreement, "
+                "and received an invitation to join the Plone GitHub organization, "
```
Comment (stevepiercy) [inline src/mr.roboto/src/mr/roboto/subscriber.py:281]:
```suggestion
                "and received and accepted an invitation to join the Plone GitHub organization, "
```

File: src/mr.roboto/src/mr/roboto/subscriber.py [hunk 4]
```diff
@@ -272,6 +276,11 @@ def run(self):
...
+                "but your emails are not known, then you might need to add "
```
Comment (stevepiercy) [inline src/mr.roboto/src/mr/roboto/subscriber.py:282]: Remove redundant clause. I was in a rush.

```suggestion
                "then you might need to add "
```

**Review by stevepiercy — APPROVED** [review]
This time I took more care. I think this is good. I'll push the commits and merge. Thank you for finding and fixing this, as it has bugged me for years!

**Comment by davisagli** [comment]
@gforcada @mauritsvanrees Could you please either deploy this or remind me how to do that?

**Comment by mauritsvanrees** [comment]
Sorry, I don't know.

**Comment by gforcada** [comment]
🙌🏾 that's me, until we find a way to automate its deployment 😓

**Comment by gforcada** [comment]
Deployed!

**Pull request merged**